gps/index.html

```diff
@@ -10,27 +10,27 @@
 date: 2018-04-09
 categories: []
 ---
-<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What is a Reference Network</h2>
+<h2 class="text-left"><i class="fas fa-fw fa-question"></i> What Is a Reference Network?</h2>
 <img class="inline-text-right outline" src="images/turn.png" alt="TURN GPS Network">
-<p>A GPS network is a high precision Global Navigation Satellite System (GNSS) providing real time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real time high accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet, that analyzes the input and adjusts and corrects the data to provide the most accurate locations usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
+<p>A GPS network is a high-precision Global Navigation Satellite System (GNSS) providing real-time corrections and data for post processing. A GNSS network consists of permanently located GPS receivers, installed across an area, that generate real-time, high-accuracy GPS positioning. Basically, all of the permanently installed GPS receivers send their data to a central system via the internet. This central system analyzes the input and adjusts and corrects the data to provide the most accurate locations, usually down to centimeter level. Our networks use the Trimble PIVOT VRS Platform. The generated data is sent through a wireless connection using the Networked Transport of RTCM via Internet Protocol (NTRIP).</p>
```

Comment:
"Our networks use the Trimble PIVOT VRS Platform.": Do we need to define "PIVOT VRS"? 

And what do we use the platform for? (i.e., Do we mean "Our networks use the Trimble PIVOT VRS Platform to analyze the GPS input."? What do we mean here?)

Reply:
I don't think we need to define the vendor. And yes our network is an implementation of the pivot vrs platform. The platform has all the intelligence to do the corrections. We just add the gps base stations and keep it all up and running.

Reply:
Left this as is